src/css/header.css

```diff
@@ -28,3 +29,36 @@
     border-bottom: 2px solid var(--brand-green);
     opacity: 1;
 }
+
+/*A suggestion of header adjustment below*/
+
+
+/* .navigation {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+} */
+
+.navigation__logo {
+    margin-top: 26.91px;
+    margin-left: 124.78px;
+    height: 17.38px;
+    width: 132.86px;
+}
+
+.navigation__button {
+    /* grid-column-start: right; */
```

Comment:
please don't push comments like this. This code is a candidate to enter the master and should be done well. Your comment can bring harm to the code.